homeassistant/components/spotify/media_player.py

```diff
@@ -373,7 +373,7 @@ def play_media(
                 raise ValueError(
                     f"Media type {media_type} is not supported when enqueue is ADD"
                 )
-            return self.data.client.add_to_queue(media_id, kwargs.get("device_id"))
+            self.data.client.add_to_queue(media_id, kwargs.get("device_id"))
```

Comment:
This is causing a behavioral change, was that intentional?

Reply:
According to the function def, the `play_media` function should return None, so the return value should not be needed.

https://github.com/home-assistant/core/blob/2b2270a05dd62b44724a8289c019985e467fb48c/homeassistant/components/spotify/media_player.py#L336-L338

Reply:
The missing return causes the next statement to be executes, which previously was not the case.

Reply:
Oh, thanks, that was an oversight from me.